CHANGELOG.md

```diff
@@ -3,6 +3,91 @@ All notable changes to this project will be documented in this file.
+- fixed behavior of `assert` and `printf` for OpenMP offloading targets #1351
+- `ALPAKA_STATIC_ACC_MEM_CONSTANT` now works correctly for clang-CUDA as well as HIP #1386
+- The OpenMP 5 and OpenACC back-ends now correctly pass the parameters as a `is_trivially_copyable` type #1387
+- Fixed `alpaka_compiler_option` checking for the wrong variable name #1392
+- Fixed a bug in the HIP back-end's peer-to-peer `memcpy` implementation #1400
+- The CMake function `alpaka_compiler_option` has been turned into a macro which solves parameter scope issues #1401
+- Fixed compilation error occuring with CUDA >= 11.3 #1404
+- the `-pthread` flag is now correctly passed to the (host) compiler and linker #1420
+- alpaka now correctly sets the CUDA host compiler #1423
+- alpaka's headers are now treated as CMake `SYSTEM` headers so that internal warnings no longer annoy users #1451
+- Projects using alpaka can now set `ALPAKA_CXX_STANDARD` without alpaka ignoring this #1463
+
+### New Features:
+- alpaka now supports the Philox random number generator #1319
+- alpaka's kernel language now supports memory fences (a.k.a thread fences) #1379
+- `alpaka::Vec` now supports structured bindings #1393
+- The OpenMP 5 and OpenACC back-ends now support statically mapped memory #1394
+- alpaka now has factory methods for creating memory views #1398
+- If OpenMP >= 5.1 is supported the back-end makes use of `atomic capture compare` #1411
+- alpaka now experimentally supports accessors instead of pointers to access memory #1433
+- added wrapper for CUDA's native vector types so that they may be handled like arrays #1435
+- added function for **exact** floating point comparisons #1440
+- added portable implementations of random number distributions (default: `TinyMersenneTwister`) #1444
+- added new math functions: `isnan`, `isinf`, `isfinite` #1446
+- new type trait that removes `__restrict__` from pointers #1474
+
+### Misc:
+- if TBB is enabled CMake is now able to pick up both oneAPI TBB and legacy TBB #1329
+- Eclipse project files are no longer tracked by git #1347
+- OpenACC atomics are now well-defined #1358
+- headers in `alpaka/test` are now installable #1360
+- HIP no longer receives special treatment inside `alpaka_add_library` #1410
+- Removed unnecessary annotations on default constructors #1416
+- Removed unnecessary defaulted and deleted special member functions #1418
+- Removed unnecessary `explicit` specifiers #1419
+- simplified implementation of `ALPAKA_UNROLL` #1437
+- math traits have been (internally) simplified #1457
+- Visual Studio project files are no longer tracked by git #1464
+- The alpaka CMake project now enables the `CXX` language by default everywhere (previously some test cases would enable `C`) #1470
+
+### Breaking Changes:
+- legacy TBB support is deprecated. alpaka will move to oneTBB in 0.9 #1329
+- HIP + nvcc is no longer supported #1337
+- `hipcc` is no longer supported as compiler for HIP. Use HIP's clang instead #1338
```

Comment:
`hipcc` is still supported but `native clang can be used too.
The title must be refrased.